ext/Devel-Peek/Peek.pm

```diff
@@ -3,14 +3,13 @@
 
 package Devel::Peek;
 
-$VERSION = '1.30';
+$VERSION = '1.31';
 $XS_VERSION = $VERSION;
 $VERSION = eval $VERSION;
 
 require Exporter;
```

Comment:
This doesn't import `&import`?

Reply:
`Devel::Peek` jumps directly to `&Exporter::import`, and doesn't need the inheritance.